careerfairsystems/arkad-app-2018
# Fixes #67 Optimize overview map

## Change description

Complete overview map and all house maps added. 

## How to verify
Click on all the buildings, check that the correct map is shown. Select the maps from the dropdown and check that the correct map is displayed.

## Issues fixed

Fixes #67
